.changeset/selfish-doors-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+fix: use non-existent package name for virtual module
```

Comment:
We should use a changeset that indicates what user-visible change this causes, which this does not. E.g., something about fixing a show-stopping regression from the previous version causing all builds to fail.

Reply:
```suggestion
fix: use non-existent package name for virtual module to allow `vite build` to succeed
```

Reply:
```suggestion
fix: solve `missing "./paths" specifier in "@sveltejs/kit" package` error occurring in all projects
```